src/Occurences.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+public class Occurences {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Enter a string : ");
+        String str = scan.nextLine();
```

Comment:
instead of all from line 5 to 7 you can just use String str = "Hello England"; Try to stop using scanner